pkg/opencontainer/default.go

```diff
@@ -0,0 +1,150 @@
+package opencontainer
+
+// DefaultRuntimeSpec is the default template for running Linux containers.
+var DefaultRuntimeSpec = LinuxRuntimeSpec{
```

Comment:
The defaults don't exist in the upstream specs repository.  Presumably, something like this is encoded in _runc_'s source code.  The configuration here is based on the output of the `runc spec` command.